app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :likes
+  has_many :comments
+
+  def return_last_three_posts
+    posts.last(3)
+  end
```

Comment:
- [ ] The post should be in descending order.

`posts.order('created_at Desc').limit(3)`